content/team/_index.md

```diff
@@ -89,6 +89,7 @@ The SkyWalking team is comprised of Members and Contributors, and the growth has
 | Ming Wen          | wenming     |                                                  |
 | Puguang Yang      | ypg         |                                                  |
 | Qiang Li          | liqiang     |                                                  |
+| Qiang Xu          | xuqiang     |                                                  |
 | Ruixian Wang      | ax1an       | <i class="iconfont icon-twitter"></i> [Ax1anRISE](https://twitter.com/Ax1anRISE)       |
```

Comment:
This should be moved to the right side table. The left side is one more than right already.

<img width="1221" alt="image" src="https://github.com/apache/skywalking-website/assets/5441976/d9a28436-e7b7-4438-b9d1-a8599db95568">

Reply:
so, i should keep my in left table and move Ruixian Wang to right table to keep alphabetical order

Reply:
Yes.

Reply:
ok，got it